caffe2/utils/threadpool/pthreadpool-cpp.cc

```diff
@@ -2,8 +2,42 @@
 #include <caffe2/utils/threadpool/thread_pool_guard.h>
 #include <c10/util/Exception.h>
 
+#include <atomic>
+
+namespace {
+// Number of threads in the thread-pool before fork
+std::atomic<int> num_threads_before_fork{-1};
```

Comment:
I literally copy-pasted this from `ParallelOpenMP.cpp`. I'm not sure but I think a simple `size_t` would have sufficed.